File: cakk-api/src/main/java/com/cakk/api/dto/response/shop/CakeShopInfoResponse.java
package com.cakk.api.dto.response.shop;

import java.util.List;

import com.cakk.domain.dto.param.shop.CakeShopOperationParam;

public record CakeShopInfoResponse(
	String shopAddress,
	Double latitude,
	Double longitude,
	List<CakeShopOperationParam> shopOperationDays
) {
}

File: cakk-core/src/main/kotlin/com/cakk/core/dto/response/shop/CakeShopInfoResponse.kt
package com.cakk.core.dto.response.shop

import com.cakk.infrastructure.persistence.param.shop.CakeShopOperationParam

data class CakeShopInfoResponse(
    val shopAddress: String,
    val latitude: Double,
    val longitude: Double,
    val shopOperationDays: List<com.cakk.infrastructure.persistence.param.shop.CakeShopOperationParam>
)
